videos/tasks.py

```diff
@@ -151,7 +151,7 @@ def update_youtube_statuses():
                     set_dict_field(
                         resource.metadata,
                         settings.YT_FIELD_THUMBNAIL,
-                        f"https://img.youtube.com/vi/{video_file.destination_id}/0.jpg",
+                        f"https://img.youtube.com/vi/{video_file.destination_id}/default.jpg",
```

Comment:
`https://img.youtube.com/vi/` and `default.jpg` is being repeated in different places, I think we should move it to constants.
Reviewer can share his/her thoughts.

Reply:
Agreed, maybe something like `YT_IMG_PREFIX` and `YT_IMG_SUFFIX`

Reply:
Sure
How about if we keep one constant and use string formatter, something like this:
```
YT_THUMBNAIL_IMG = "https://img.youtube.com/vi/{video_id}/default.jpg"
YT_THUMBNAIL_IMG.format(video_id = video_file.destination_id)
```

What do you suggest?